core-primitives/attestation-handler/src/cert.rs

```diff
@@ -15,20 +15,27 @@
 
 */
 
-use super::CERTEXPIRYDAYS;
+#[cfg(all(not(feature = "std"), feature = "sgx"))]
+use crate::sgx_reexport_prelude::*;
+
 use crate::{Error as EnclaveError, Result as EnclaveResult};
 use arrayvec::ArrayVec;
-use bit_vec::BitVec;
-use chrono::{prelude::*, Duration, TimeZone, Utc as TzUtc};
 use itertools::Itertools;
 use itp_ocall_api::EnclaveAttestationOCallApi;
 use log::*;
-use num_bigint::BigUint;
+
```

Comment:
Is there a specific reason for this newline? it will prevent `rustfmt` from formatting, i.e. ordering, the uses correctly

Reply:
Not at all. I will remove it. I was not aware that this affects `rustfmt`. Interesting.